ocs/utils.py

```diff
@@ -606,3 +606,37 @@ def create_oc_resource(
         f.write(template)
     log.info(f"Creating rook resource from {template_name}")
     run_cmd(f"oc create -f {cfg_file}")
+
+
+def apply_oc_resource(
+    template_name,
+    rook_data,
+    cluster_path,
+    _templating,
+    template_dir="ocs-deployment",
+):
+    """
+    Apply an oc resource after rendering the specified template with
+    the rook data from cluster_conf.
+
+    Args:
+        template_name (str): Name of the ocs-deployment config template
+        rook_data (dict): Rook specific config from cluster_conf
+        cluster_path (str): Path to cluster directory, where files will be
+            written
+        _templating (Templating): Object of Templating class used for
+            templating
+        template_dir (str): Directory under templates dir where template
+            exists
+    """
+    base_name = template_name.split('.')[0]
+    template_path = os.path.join(template_dir, template_name)
+    template = _templating.render_template(
+        template_path,
+        rook_data.get(base_name, {})
+    )
+    cfg_file = os.path.join(cluster_path, template_name)
+    with open(cfg_file, "w") as f:
+        f.write(template)
+    log.info(f"Applying rook resource from {template_name}")
+    run_cmd(f"oc apply -f {cfg_file}")
```

Comment:
Can you try using OCP.apply({cfg_file})?

Reply:
Done.